Makefile

```diff
@@ -24,7 +30,7 @@ else
 	./ginkgo -v functional/ -- -runtime=${RUNTIME} -timeout=${TIMEOUT}
 endif
 
-integration: ginkgo
+docker: ginkgo
```

Comment:
I like this change :smile:

Reply:
please update `.PHONY`, remove `integration` and add `docker`

Reply:
ok. update asap.

Reply:
Still waiting for that `.PHONY` update s/integration/docker/ change afaict. Are there other pending changes on this PR @Pennyzct ?

Reply:
@grahamwhaley  https://github.com/kata-containers/tests/pull/514/files#diff-b67911656ef5d18c4ae36cb6741b7965R93

Reply:
Hmm, oh. I wasn't seeing that in my github file view just now.... thx @devimc